application/README.md

```diff
@@ -0,0 +1,46 @@
+# The Prio Server Runtime
+
+The application folder contains all of the components that are necessary for
```

Comment:
I would suggest a more meaningful directory name than "application". Perhaps simply "batch-processor"?

Reply:
The `application` folder is now `processor`.